uobtheatre/venues/schema.py

```diff
@@ -19,6 +20,7 @@ class Meta:
 
 class VenueNode(GrapheneImageMixin, DjangoObjectType):
     image = GrapheneImageField(GrapheneImageFieldNode)
+    address = graphene.Field(AddressNode)
```

Comment:
In theory, this should happen automagically
```suggestion
```

Reply:
For context, it seems that `DjangoObjectType` will automatically add any field which is either a simple type (like a string or int) or if its a custom model it will add it if a `DjangoObjectType` exists for that model. So in theory simply creating the `AddressNode` will make the address available